### DISCUSSION TITLE
LoRaWAN Protocol Support

### DISCUSSION THREAD

**Discussion by mgodmere** [eclipse-zenoh/roadmap#173] [Ideas and new features]
LoRaWAN is on the rise as a critical IoT protocol.  Is this planned to be added to Zenoh pico in the future?

**Comment by Mallets** [comment]
Hello, 

This is not planned so far.
If you'd like to speed up the roadmap you can contact [ZettaScale](https://www.zettascale.tech/contact-us/) for support.
